images/scripts/lib/build-deps.sh

```diff
@@ -41,4 +41,12 @@ case "$OS_VER" in
     *) ;;
 esac
 
+# pull nodejs-devel on Fedora for compliance with the guidelines on using nodejs modules:
+# https://docs.fedoraproject.org/en-US/packaging-guidelines/Node.js/#_buildrequires
+# pull it also on RHEL 8 & derivatives, as it exists there; it does not exist in newer RHELs
+case "$OS_VER" in
+    fedora*|rhel*8|centos*8) EXTRA_DEPS="$EXTRA_DEPS nodejs-devel" ;;
```

Comment:
Let's please just limit it to Fedora then. I don't see a need to install it on RHEL 8.